site/cypress/e2e/pronouns_change.spec.js

```diff
@@ -0,0 +1,76 @@
+
+describe('Tests cases abut changing user pronouns', () => {
+
+    let oldPronouns = '';
+
+    it('Tests changing pronouns for student', () => {
```

Comment:
Even though it seems good to have multiple points being tested, since these tests are using the same values, and the tests 'rely' on each other passing, it would be better to combine them into one bigger test. This would be as easy as just removing the separate 'it' statements, and putting the description as a comment, and explicitly logging out of each user before logging in as a different one.

Reply:
It logs in for each test already. `cy.login()` automatically logs out and then logs in again (or at least, it does in these test cases). 

Combining the test would have the issue of making it harder to know what failed during the test

Reply:
Should be addressed in latest commit. Let me know if you have any questions/concerns